azext_iot/_help.py

```diff
@@ -1096,6 +1101,14 @@
         --target-condition "tags.building=9 and tags.environment='test'"
         --metrics metrics_content.json
         --layered
+    - name: Create a layered deployment with an alternative input style of labels and metrics (powershell syntax example)
```

Comment:
The change in syntax for custom labels and custom metrics is meant to simplify and be shell agnostic, you are no longer dealing with json. So the `(powershell syntax example)` segment doesn't make sense.

Reply:
we could say `(shell agnostic)`